pkg/nuctl/command/import.go

```diff
@@ -114,17 +116,37 @@ func (i *importCommandeer) importFunction(ctx context.Context, functionConfig *f
 func (i *importCommandeer) importFunctions(ctx context.Context,
 	functionConfigs map[string]*functionconfig.Config,
 	project *platform.ProjectConfig) error {
-	errGroup, _ := errgroup.WithContext(ctx, i.rootCommandeer.loggerInstance)
-
+	importFailed := atomic.Bool{}
 	i.rootCommandeer.loggerInstance.DebugWithCtx(ctx, "Importing functions", "functions", functionConfigs)
+	wg := sync.WaitGroup{}
 	for _, functionConfig := range functionConfigs {
-		functionConfig := functionConfig // https://golang.org/doc/faq#closures_and_goroutines
-		errGroup.Go("Import function", func() error {
-			return i.importFunction(ctx, functionConfig, project)
-		})
+		wg.Add(1)
+		go func(function *functionconfig.Config) {
+			i.rootCommandeer.loggerInstance.DebugWithCtx(ctx, "Importing function",
+				"function", function.Meta.Name,
+				"project", project.Meta.Name)
+			if err := i.importFunction(ctx, function, project); err != nil {
+				if !importFailed.Load() {
+					importFailed.Store(true)
+				}
+				i.rootCommandeer.loggerInstance.ErrorWithCtx(ctx, "Failed to import function",
+					"function", function.Meta.Name,
+					"project", project.Meta.Name,
+					"error", err)
+			} else {
+				i.rootCommandeer.loggerInstance.DebugWithCtx(ctx, "Function was imported successfully",
+					"function", function.Meta.Name,
+					"project", project.Meta.Name)
+			}
+			wg.Done()
+		}(functionConfig)
 	}
+	wg.Wait()
 
-	return errGroup.Wait()
+	if importFailed.Load() {
+		return errors.New("Import failed for some of the functions")
+	}
```

Comment:
this needs to be more indicative. at least provide function names

Reply:
@liranbg then we will logs them twice - firstly right from goroutine and secondly from here

Reply:
@rokatyy It's not that bad to log that again, because with many functions the logs can be noisy and it can be easy to miss warnings/errors.

a single log line at the end can be very useful.

Reply:
@TomerShor Agreed. I've added TODO for this. The plan is to create a custom structure for `importReport` in the following PR which will contain methods like `Print` and `SaveToFile`.